xc/device.ml

```diff
@@ -2083,14 +2083,25 @@ module Backend = struct
     (** Implementation of the Vbd functions that use the dispatcher for the qemu-upstream-compat backend *)
     module Vbd = struct
 
+      let cd_of devid =
+        devid
+        |> Device_number.of_xenstore_key
```

Comment:
I was looking where this module is defined, it is here: https://github.com/xapi-project/xcp-idl/blob/master/xen/device_number.mli